src/types/Stream.d.ts

```diff
@@ -1,10 +1,27 @@
 import Client from './Client'
+import { ExprArg } from './query'
 
 export interface Subscription {
-  client: Client
+  client: StreamClient
   dispatcher: EventDispatcher
 }
 
 export interface EventDispatcher {
   allowedEvents: string[]
```

Comment:
This looks too generic as well, we should say specifically the events.

Reply:
I don't think it's worth documenting the internal types. Users don't have access to it.